index.html

```diff
@@ -1,242 +1,466 @@
-              Get in touch
-            </button>
-          </form>
+      </section>
+      <section class="section-about margin-section color">
+        <div class="about">
+          <div class="about-class">
+            <h2 class="heading-tertiary">
+              About <br />
+              myself
+            </h2>
+            <p class="about-info">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h2 class="heading-secondary">Let's Connect</h2>
+            <ul class="social-icons">
+              <li>
+                <a href="#">
+                  <i class="social-icon fa-brands fa-twitter"></i>
+                </a>
+              </li>
+
+              <li>
+                <a href="#">
+                  <i class="social-icon fa-brands fa-linkedin"></i>
+                </a>
+              </li>
+
+              <li>
+                <a href="#">
+                  <i class="social-icon fa-solid fa-envelope"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <i class="social-icon fa-brands fa-github"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <i class="social-icon fa-regular fa-hand-peace"></i>
+                </a>
+              </li>
+            </ul>
+
+            <a class="btn" href="#">Get my resume</a>
+          </div>
+          <div class="languages">
+            <div class="subheading margin-section">
+              <p class="subheader">Languages</p>
+              <img
+                class="subheader-icon"
+                src="Images/down-cursor.png"
+                alt="an arrow pointing down"
+              />
+            </div>
+            <ul class="languages-used">
+              <li class="language-used">
+                <img src="Images/Ellipse 1.png" alt="javascript logo" />
+
+                <span>Javascript</span>
+              </li>
+
+              <li class="language-used">
+                <img src="Images/Ellipse 1 (1).png" alt="javascript logo" />
+
+                <span>HTML</span>
+              </li>
+
+              <li class="language-used">
+                <img src="Images/Ellipse 1 (2).png" alt="javascript logo" />
+
+                <span>CSS</span>
+              </li>
+            </ul>
+            <div class="frameworks-skills">
+              <p class="subheader">Frameworks</p>
+              <img src="images/side-arrow.png" alt="An arrow pointing right" />
+            </div>
+
+            <div class="frameworks-skills">
+              <p class="subheader">Skills</p>
+              <img src="images/side-arrow.png" alt="An arrow pointing right" />
+            </div>
```

Comment:
[OPTIONAL] Great job @tanveerisonline 👍. Kindly check your image path, the arrow is not showing.
![arro](https://user-images.githubusercontent.com/73854999/200118061-90fa60cd-17c4-40a9-b8ac-ab4c9c61a640.PNG)

Reply:
hi but i have checked my project in 4 different browsers and it is working fine may be from your side their is some issue